### PR TITLE
[Accounts] Generate Users auth (1)

### PR DESCRIPTION
**Pull request by rafalskorupa**

### CODE REVIEW

**Review by erszcz — CHANGES_REQUESTED** [review]
Hey @rafalskorupa,
this looks good as far as autogeneration of the auth modules, but these in turn seem not to be glued together correctly, so I couldn't really finish the registration process. Could you look into the issues mentioned inline?

File: priv/repo/migrations/20240304080229_create_users_auth_tables.exs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+defmodule ChirpCockroach.Repo.Migrations.CreateUsersAuthTables do
+  use Ecto.Migration
+
+  def change do
+
+    create table(:users) do
```
[the remark this answers is not in the document]
Comment (rafalskorupa) [reply priv/repo/migrations/20240304080229_create_users_auth_tables.exs:6]: Migration fixed && checked with resetting database - I did some rebasing wrong 😅

File: config/config.exs [hunk 2]
```diff
@@ -10,6 +10,12 @@ import Config
 config :chirp_cockroach,
   ecto_repos: [ChirpCockroach.Repo]
 
+
+config :chirp_cockroach, ChirpCockroach.Mailer, adapter: Swoosh.Adapters.Local
```
[the remark this answers is not in the document]
Comment (rafalskorupa) [reply config/config.exs:14]: Confirmation is not required for using `users` - mailbox preview added, but confirming account is not currently needed in the app.

[the remark this answers is not in the document]
Comment (erszcz) [reply config/config.exs:14]: Ok, I couldn't check if confirmation is required because of the other issue. Good to know 👍

**Review by erszcz — APPROVED** [review]

**Comment by erszcz** [comment]
Looks good! Thanks, @rafalskorupa!

**Pull request merged**